openpype/hosts/houdini/plugins/create/create_redshift_rop.py

```diff
@@ -3,12 +3,12 @@
 import hou  # noqa
 
 from openpype.hosts.houdini.api import plugin
-from openpype.pipeline import CreatedInstance
```

Comment:
If it is not included, please remove the comment in line 31 `# type: CreatedInstance`. thanks!